### PR TITLE
Tinydns additions

### PR DESCRIPTION
**Pull request by adhawkins**
Add support for TinyDNS TXT (#340) and AAAA (#342) records.

First pull request, so comments appreciated.

### CODE REVIEW

File: octodns/source/tinydns.py [hunk 1]
```diff
@@ -45,6 +46,37 @@ def _data_for_A(self, _type, records):
             'values': values,
         }
 
+    def _data_for_AAAA(self, _type, records):
+        values = []
+        for record in records:
+            values.append(u":".join(textwrap.wrap(record[0], 4)))
```
Comment (ross) [inline octodns/source/tinydns.py]: What's going on with the textwrap stuff here? I'm not grokking what's happening here. It'll definitely need an explanatory comment if it's required, but it's fine to start discussion here first.

Comment (adhawkins) [reply octodns/source/tinydns.py]: TinyDNS files have the ipv6 address written in full, but with the colons removed. This just inserts a colon every 4th character to make the address correct.

Will add a comment when I get chance.

Comment (ross) [reply octodns/source/tinydns.py]: Ugh. OK. Please add a comment to that effect in the code as it's definitely non-obvious/standard.

Comment (adhawkins) [reply octodns/source/tinydns.py]: Will do. Or can you suggest a clearer and perhaps more Pythonic way of doing it?

Comment (ross) [reply octodns/source/tinydns.py]: I can't think of anything cleaner and it sounds like `textwrap.wrap` makes sense here. The suprising/weird part to me is that TinyDNS doesn't have `:`. 

> TinyDNS files have the ipv6 address written in full, but with the colons removed. This just inserts a colon every 4th character to make the address correct.

Will work as the comment.

Comment (adhawkins) [reply octodns/source/tinydns.py]: Comment added. Let me know if you need anything further.

**Pull request merged**